server/modules/api_endpoints/ProtubeApi.js

```diff
@@ -58,6 +59,14 @@ this.protubeApi.post("/updateadmin", async function (req, res) {
   return res.send({ success: enums.SUCCESS });
 });
 
+// Endpoint to skip a song
+this.protubeApi.post("/skipsong", function (req, res) {
+  try {
+    playNextVideo();
+  } catch {}
+  return res.send({ success: enums.SUCCESS });
```

Comment:
Maybe you should return an 500 error when an exception occurs?
And you should probably specify which error you are catching.
And if it is and critical error you also shouldn't return a success

Reply:
It throws an error if it cant skip the queue (if its empty for example).
Depends on how you want to catch the errors, do you want them through status codes or a code 200 with a json

Reply:
Yeah it might be nicer to do it with the json success: false and if you have the error also give the error message